config/7.10.0/common.yaml

```diff
@@ -116,8 +116,12 @@ console:
                     value: "[[.KeyStorePassword]]"
                   - name: WORKBENCH_ROUTE_NAME
                     value: "[[.ApplicationName]]-[[.Console.Name]]"
+                  - name: KUBERNETES_NAMESPACE
+                    value: "${KUBERNETES_NAMESPACE}"
+                  - name: KUBERNETES_LABELS
+                    value: "${KUBERNETES_LABELS}"
                   - name: JGROUPS_PING_PROTOCOL
-                    value: "openshift.DNS_PING"
+                    value: "kubernetes.KUBE_PING"
                   - name: OPENSHIFT_DNS_PING_SERVICE_NAME
                     value: "[[.ApplicationName]]-[[.Console.Name]]-ping"
```

Comment:
shouldn't it be remove as well?
have you confirmed if we can delete the ping port and service?

Reply:
I'll try to see if the amq and datagrid works correclty if the JGROUPS_PING_PROTOCOL is chaged but the other envs are preserved (OPENSHIFT_DNS_PING_SERVICE_NAME and OPENSHIFT_DNS_PING_SERVICE_PORT)